Changed default FPE_HOSTNAME to 'observatorysim'.

mlv opened the pull request:
It can still be overwritten by os.environ['FPE_HOSTNAME']
Fixed ping to use self.FPE_HOSTNAME
Added global DEFAULT_TFTP_HOSTNAME and using that for TFTP (since it can't use the proxy)

xcthulhu commented:
This is a breaking change; without a fallback to the old behavior this just makes a maintenance headache for myself.  I'm postponing merging this.

In the interim, I think it's just easier to set environment variables.

mlv commented:
Did you notice that your environment variable change doesn't work? I fixed that, too. Also, you can't use the same hostname for 5555/5556 and for tftp.

The fallback is to use $FPE_HOSTNAME (which I fixed)
